source/root_output.f90

```diff
@@ -307,7 +307,7 @@ subroutine root_daten(inLine,iErr)
 
   call chr_split(inLine,lnSplit,nSplit,spErr)
   if(spErr) then
-    write(lout,"(a)") "ZIPF> ERROR Failed to parse input line."
+    write(lerr,"(a)") "ZIPF> ERROR Failed to parse input line."
```

Comment:
Uhm, while the code may have been copied from ZIPF, we're not in ZIPF...

Reply:
James' stuff. I'll fix that one anyway.